src/views/cards/l10n-static.json

```diff
@@ -1,14 +1,15 @@
 {
-    "cards.starterLink": "/pdfs/cards/Scratch2Cards.pdf",
-    "cards.nameLink": "/pdfs/cards/AnimateYourNameCards.pdf",
-    "cards.flyLink": "/pdfs/cards/FlyCards.pdf",
-    "cards.raceLink": "/pdfs/cards/RaceGameCards.pdf",
-    "cards.musicLink": "/pdfs/cards/MusicCards.pdf",
-    "cards.hideLink": "/pdfs/cards/Hide-and-Seek-Cards.pdf",
-    "cards.storyLink": "/pdfs/cards/StoryCards.pdf",
-    "cards.dressupLink": "/pdfs/cards/DressupCards.pdf",
-    "cards.pongLink": "/pdfs/cards/PongCards.pdf",
-    "cards.danceLink": "/pdfs/cards/DanceCards.pdf",
-    "cards.catchLink": "/pdfs/cards/CatchCards.pdf",
-    "cards.petLink": "/pdfs/cards/PetCards.pdf"
-}
+    "cards.Scratch2CardsLink": "https:/resources.scratch.mit.edu/www/cards/en/Scratch2Cards.pdf",
```

Comment:
Just goes to show that the value part of l10n-static is no longer needed. They all get set from the resources.scratch.mit.edu generated localized-urls.